src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -566,17 +567,7 @@ public static void ConvertTime_DateTime_PerthRules()
 
             time1utc = new DateTime(2006, 12, 31, 15, 59, 59, DateTimeKind.Utc);
             time1 = new DateTime(2006, 12, 31, 15, 59, 59);
-            if (s_isWindows)
-            {
-                // ambiguous time between rules
-                // this is not ideal, but the way it works
```

Comment:
This fix is also fixing an existing bug for Perth, right?

Reply:
That is right. it is was similar case as Fjii we are fixing